boards/xtensa/heltec_wifi_lora32_v2/heltec_wifi_lora32_v2.dts

```diff
@@ -5,7 +5,7 @@
  */
 /dts-v1/;
 
-#include <espressif/esp32/esp32_wrover_e_n4r8.dtsi>
+#include <espressif/esp32/esp32_d0wd_v3.dtsi>
```

Comment:
Is it a fixup of previous commit?

Reply:
This is fixup since `heltec_wifi_lora32_v2` does not use SIP.